addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/client/RpcClient.java

```diff
@@ -79,6 +79,9 @@ public void init(HmInterface hmInterface, String clientId) throws IOException {
         RpcRequest<T> request = createRpcRequest("init");
         request.addArg(getRpcCallbackUrl());
         request.addArg(clientId);
+        if (config.getGatewayInfo().isHomegear()) {
+            request.addArg(new Integer(0x20));
+        }
```

Comment:
What's the reason for this addition? According to Saytha (Homegear maintainer), this is should include the binary flag for BinRPC messages: https://forum.homegear.eu/t/nochmal-openhab-rpc-problem/1829/7

Reply:
@pauxus https://forum.homegear.eu/t/pingpong-problem/1807

Reply:
see https://forum.homegear.eu/t/nochmal-openhab-rpc-problem/1829/10